guides/context-helpers.md

```diff
@@ -253,15 +253,13 @@ Try these exercises to help you further your understanding of context helpers!
 
 Your JSON data is badly-formatted, as seen in the sample. Write a helper to reformat this data to make the template work.
 
-```
+<dust-tutorial>
```

Comment:
Can you add a `templatename` attribute to each of these (and `showtemplatename="true"` on the `dust-demo-template` tag)